package.json

```diff
@@ -18,7 +18,7 @@
     "mkdirp": "^0.5.1",
     "noms": "0.0.0",
     "through2": "^2.0.1",
-    "yargs": "^11.0.0"
+    "yargs": "^13.2.4"
```

Comment:
The biggest difference in the upgrade, from what I can tell, is how `args.argv` behaves. Here's argv in `^11`:

```console
./copyfiles /tmp/a.js /tmp/b.js -u1
{ _: [ '/tmp/a.js', '/tmp/b.js' ],
  version: false,
  v: false,
  a: false,
  all: false,
  f: false,
  flat: false,
  E: false,
  error: false,
  V: false,
  verbose: false,
  s: false,
  soft: false,
  help: false,
  h: false,
  u: 1,
  up: 1,
  '$0': 'copyfiles',
  e: undefined }
```

and in `^13`:

```console
./copyfiles /tmp/a.js /tmp/b.js -u1
{ _: [ '/tmp/a.js', '/tmp/b.js' ],
  u: 1,
  up: 1,
  '$0': 'copyfiles' }
```

Glancing at the usage, I don't think that'll be a problem (since it'll be `undefined` instead of `false`).

Here's the [full changelog](https://github.com/yargs/yargs/blob/master/CHANGELOG.md).